wger/exercises/fixtures/muscles.json

```diff
@@ -126,5 +126,29 @@
             "is_front": false, 
             "name": "Triceps brachii"
         }
+    },
+    {
+        "pk": 17,
+        "model": "exercises.muscle",
+        "fields": {
+            "is_front": false,
+            "name": "Lateral Deltoid"
+        }
+    },
+    {
+        "pk": 20,
+        "model": "exercises.muscle",
+        "fields": {
+            "is_front": true,
+            "name": "Forearms"
+        }
+    },
+    {
+        "pk": 19,
+        "model": "exercises.muscle",
+        "fields": {
+            "is_front": false,
+            "name": "Posterious Deltoid"
```

Comment:
I am not able to see this muscle `Posterious Deltoid` in the diagram. Was it added?

Reply:
Apparently, I had not added the muscles. You should now be able to see them.